packages/veritone-widgets/rollup.config.base.js

```diff
@@ -61,7 +61,22 @@ export default {
       // modulesOnly: true
     }),
     commonjs({
-      include: ['../../node_modules/**', 'node_modules/**', '../**']
+      include: ['../../node_modules/**', 'node_modules/**', '../**'],
+      exclude: [
+        'node_modules/process-es6/**'
+      ],
+      namedExports: {
```

Comment:
Is this a hack :)

Reply:
i'm not sure but it fixed roll-up issue when building the app :D